doc/introduction/basic_usage/basic_usage.rst

```diff
@@ -0,0 +1,109 @@
+Basic Usage
+=============
+
+PaddlePaddle is a deep learning platform open-sourced by Baidu. With PaddlePaddle, you can easily train a classic neural network within a couple lines of configuration, or you can build sophisticated models that provide state-of-the-art performance on difficult learning tasks like sentiment analysis, machine translation, image caption and so on.
+
+1. A Classic Problem
+---------------------
+
+Now, to give you a hint of what using PaddlePaddle looks like, let's start with a fundamental learning problem - `simple linear regression <https://en.wikipedia.org/wiki/Simple_linear_regression>`_: you have observed a set of two-dimensional data points of ``X`` and ``Y``, where ``X`` is an explanatory variable and ``Y`` is corresponding dependent variable, and you want to recover the underlying correlation between ``X`` and ``Y``. Linear regression can be used in many practical scenarios. For example, ``X`` can be a variable about house size, and ``Y`` a variable about house price. You can build a model that captures relationship between them by observing real estate markets.
+
+2. Prepare the Data
+--------------------
+
+Suppose the true relationship can be characterized as ``Y = 2X + 0.3``, let's see how to recover this pattern only from observed data. Here is a piece of python code that feeds synthetic data to PaddlePaddle. The code is pretty self-explanatory, the only extra thing you need to add for PaddlePaddle is a definition of input data types.
```

Comment:
这一节的重点是要介绍Paddle job通常是通过 Python 写的 data provider 来读取数据的。所以第一段的第一句应该是介绍 data provider的。比如：

> A PaddlePaddle job usually loads the training data by implementing a Python *data provider*. A data provider is a Python function which is called by PaddlePaddel trainer program, so it could adapt to any data format.  We can write data provider to read from local filesystem, HDFS, databases, S3 or almost anywhere. In this example, our data provider synthesize the training data by sampling from the line Y=2X + 0.3.